test/module/irohad/ametsuchi/ametsuchi_mocks.hpp

```diff
@@ -246,6 +247,16 @@ namespace iroha {
       }
     };
 
+    class MockFlatFile : public FlatFile {
+     public:
+      MOCK_METHOD2(add, bool(Identifier id, const std::vector<uint8_t> &blob));
+      MOCK_CONST_METHOD1(get,
+                         boost::optional<std::vector<uint8_t>>(Identifier id));
+      MOCK_CONST_METHOD0(directory, std::string(void));
```

Comment:
void is redundant, similar to function args name (e.g `id` above)

Reply:
Void is used everywhere upwards, so I thought it was a good idea to stick to existing code style?